admin_guide/taints_tolerations.adoc

```diff
@@ -0,0 +1,329 @@
+[[admin-guide-taints]]
+= Taints and tolerations
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+A _taint_ allows a node to refuse to allow a pod to be scheduled unless that pod has a _toleration_ that allows the scheduling.
+
+In short, *taints* go on nodes through the node specification (NodeSpec) and *tolerations* go on pods through the pod specification (PodSpec). A taint on a node
+means _repel all pods that do not tolerate the taint_.
+
+
+A taint consists of a key, value, and effect. As a command-line argument, it is expressed as key=value:effect.
+
+.Taint components
+[cols="3a,8a",options="header"]
+|===
+
+|Resource Name |Description
+
+|`key`
+|The `key` is any string, up to 253 characters. The key must begin with a letter or number, and may contain letters, numbers, hyphens, dots, and underscores.
+
+|`operator`
+|
+
+|`value` 
+| The `value` is any string, up to 63 characters. The value must begin with a letter or number, and may contain letters, numbers, hyphens, dots, and underscores.
+
+|`effect`
+
+|[frame=none]
+[cols="2,3"]
+!====
+!`NoSchedule`!The scheduler will not schedule a pod without this toleration.
```

Comment:
What about a sentence above the list:
"If the node has taint that contains one of the following:"